openff/toolkit/utils/toolkits.py

```diff
@@ -3353,8 +3353,8 @@ def from_smiles(
                 # set it back to zero
                 atom.SetAtomMapNum(0)
 
-        # TODO: I think UpdatePropertyCache(strict=True) is called anyway in Chem.SanitizeMol().
-        rdmol.UpdatePropertyCache(strict=False)
+        # Chem.SanitizeMol calls updatePropertyCache so we don't need to call it ourselves
+        # https://www.rdkit.org/docs/cppapi/namespaceRDKit_1_1MolOps.html#a8d831787aaf2d65d9920c37b25b476f5
```

Comment:
Ah, molecule sanitization. Any method that touches sanitization is a huge tangle of complexity so I try not to touch it too often. So this change made me a bit nervous, but I looked down the code paths in RDKit and agree that running `updatePropertyCache(strict=True)` (which happens inside `Chem.SanitizeMol` ) will not be affected by previous results arising from running with `strict=False`. I also reran our slow tests and this seems to be fine. So I think this change is good to go.